Snowflake.Data/Core/Authenticator/ExternalBrowserAuthenticator.cs

```diff
@@ -252,6 +258,8 @@ private SFRestRequest BuildAuthenticatorRestRequest(int port)
                 AccountName = session.properties[SFSessionProperty.ACCOUNT],
                 Authenticator = AUTH_NAME,
                 BrowserModeRedirectPort = port.ToString(),
+                DriverName = SFEnvironment.DriverName,
```

Comment:
this improvement is not related to the feature itself?

Reply:
It's not directly related, but I saw that other connectors like JDBC include the driver name and version in the request so I thought to include it as well in .NET for parity